lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -35,41 +35,72 @@ module RSpec
       #     # ...
       #   end
       #
+      # This cop can be customized allowed context description pattern
+      # with `AllowedPatterns`. By default, there are no checking by pattern.
+      #
+      # @example `AllowedPatterns` configuration
+      #
+      #   # .rubocop.yml
+      #   # RSpec/ContextWording:
+      #   #   AllowedPatterns:
+      #   #     - /とき$/
+      #
+      # @example
+      #   # bad
+      #   context '条件を満たす' do
+      #     # ...
+      #   end
+      #
+      #   # good
+      #   context '条件を満たすとき' do
+      #     # ...
+      #   end
+      #
       class ContextWording < Base
-        MSG = 'Start context description with %<prefixes>s.'
+        include AllowedPattern
+
+        MSG = 'Context description should match %<patterns>s.'
```

Comment:
I'm uncertain that I'm happy with this change.
From my perspective, the cognitive gap between "Start description with" and "Description should match /@$&#%&$$/" is noticeable.

Wondering if it's easy to use the old message when only `Prefixes` are defined, and the new one when `AllowedPatterns` are defined, too.
Or a mix of like "Context description should either start with %<prefixes> or match %<patterns>".